### ISSUE TITLE
iw (hebrew) is not working 

### ISSUE THREAD

**Issue by Omer7731** [kaixxx/noScribe#41]
if you choose "iw" (hebrew)  you getting an error

Error in step 3 - transcription
'iw' is not a valid language code.

need to change "is" to "he"

**Comment by kaixxx** [comment]
You are right, we are going to change this in the next version. Does it work with "auto"?

For the time beeing, there is a hack to get it working: 
- close noScribe
- open the file "config.yml" in your user settings folder (on windows: "C:\Users\<CURRENT USER NAME>\AppData\Local\noScribe\noScribe")
- change the line with "last_language..." to:  "last_language: he (hebrew)" (without quotes)
- save the file, restart noScribe
- this should work until you switch to another language. Then you'd have to edit config.yml again.

**Comment by Omer7731** [comment]
yes, its works perfectly under auto.

Btw we will be happy if in the next update, you can add 2 new features
1, srt export.
2. line limit by words or by characters.

thanks a lot.

**Comment by Omer7731** [comment]
there is an option to communicate with you by Private message? want to offer you something. 
my email is Omer7731@gmail.com (I will be very happy to receive email from you)
